app/formatters/duty_expression_formatter.rb

```diff
@@ -77,14 +81,21 @@ def format(opts = {})
           output << monetary_unit
         end
         if measurement_unit_abbreviation.present?
-          output << if opts[:formatted]
+          output << if formatted
                       "/ <abbr title='#{measurement_unit.description}'>#{measurement_unit_abbreviation}</abbr>"
                     else
                       "/ #{measurement_unit_abbreviation}"
                     end
         end
       end
-      output.join(' ').html_safe
+
+      result = output.join(' ').html_safe
+
+      if resolved_meursing_component
+        "<strong>#{result}</strong>"
+      else
+        result
+      end
     end
```

Comment:
and here is one of my annoying generic comments 😄  this method is getting longer and longer, maybe it is doing too many things? can we split his responsibility?

Reply:
Yarp, one to be completely refactored